run_tests.sh

```diff
@@ -32,4 +32,12 @@ if [ "$TEST_DIR" == "ruby" ]; then
   exit
 fi
 
+if [ "$TEST_DIR" == "rust" ]; then
+  curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
Well, that was painful to implement!  I'll have to invest a bit of time in a local setup of TravisCI so I can verify it works before pushing changes, going forward!